.travis.yml

```diff
@@ -1,13 +1,14 @@
-sudo: false
 language: node_js
-cache:
-  directories:
-    - ~/.npm
```

Comment:
I think we want to keep sido as false and the cache

Reply:
I don't understand what you mean right here 😕

Reply:
Sorry, I mean all changes above this comment should be reverted (I'm on my phone and mistyped sudo)

Reply:
`sudo` key is deprecated and `cache: npm` is the same as

```yml
cache:
  directories:
    - ~/.npm
```

So I think we can keep it this way

Reply:
Oh nice!